### PR TITLE
misc: add issue assigner bot

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
If we're ready to give it a shot this adds a bot to automatically assign issues in a round-robin fashion for initial response :)

### CODE REVIEW

File: .github/workflows/issue-assigner.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  issues:
+    types: [opened]
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: patrickhulce/issue-assigner@eeec7a1
+        with:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+          maintainers: 'patrickhulce,paulirish,connorjclark,adamraine'
```
Comment (patrickhulce) [inline .github/workflows/issue-assigner.yml:12]: if anyone else would like to be a part of this, happy to add :) 

@adrianaixba @Beytoven perhaps?

Comment (brendankenny) [reply .github/workflows/issue-assigner.yml:12]: I'm happy to take issues too

Comment (patrickhulce) [reply .github/workflows/issue-assigner.yml:12]:
```suggestion
          maintainers: 'patrickhulce,paulirish,connorjclark,adamraine,brendankenny'
```

Comment (patrickhulce) [reply .github/workflows/issue-assigner.yml]: You got it :)

**Review by adamraine — APPROVED** [review]

**Pull request merged**